docs/reference/upgrade/close-ml.asciidoc

```diff
@@ -25,8 +25,9 @@ prevent new jobs from opening, use the <<ml-set-upgrade-mode,set upgrade mode AP
 POST _ml/set_upgrade_mode?enabled=true
 --------------------------------------------------
 // CONSOLE
+// TEARDOWN
```

Comment:
I think this transforms this snippet from a test into a thing that modifies tests on this page. But there *aren't* tests on this page. So I think this just makes this snippet not run.

I think something like 

```
////////////
Take us out of upgrade mode after running any snippets on this page.

[source,js]
--------------------------------------------------
POST _ml/set_upgrade_mode?enabled=false
--------------------------------------------------
// CONSOLE
// TEARDOWN
////////////
```

Will run the snippet and always clean up.